### PR TITLE
change min rd to 250 and retune parmaters

### PR DESCRIPTION
**Pull request by yairchn**

### CODE REVIEW

File: Turbulence_PrognosticTKE.pyx [hunk 1]
```diff
@@ -1350,7 +1350,7 @@ cdef class EDMF_PrognosticTKE(ParameterizationBase):
             if self.use_const_plume_spacing:
                 self.pressure_plume_spacing[i] = self.constant_plume_spacing
             else:
-                self.pressure_plume_spacing[i] = fmax(self.aspect_ratio*self.UpdVar.updraft_top[i], 500)
+                self.pressure_plume_spacing[i] = fmax(self.aspect_ratio*self.UpdVar.updraft_top[i], 250)
```
Comment (ilopezgp) [inline Turbulence_PrognosticTKE.pyx]: Can we actually change this to 250.0? Integers and float operations are not good friends

Comment (yairchn) [reply Turbulence_PrognosticTKE.pyx]: good catch

File: tests/les_data/reduce_pycles_netcdf.py [hunk 2]
```diff
@@ -126,18 +140,26 @@ def main():
...
     massflux_h_        = np.multiply(rho_temp, massflux_h_)
-    massflux_qt_       = np.multiply(rho_temp, massflux_qt_)
+    massflux_qt_       = np.multiply(rho_temp, massflux_qt_) # already multiplied qt by 1000 above
+    massflux_u_        = np.multiply(rho_temp, massflux_u_)
+    massflux_v_        = np.multiply(rho_temp, massflux_v_)
     total_flux_h_      = np.multiply(rho_temp, total_flux_h_)
-    total_flux_qt_     = np.multiply(rho_temp, total_flux_qt_)
-    diffusive_flux_h_  = np.multiply(rho_temp, diffusive_flux_h_)
-    diffusive_flux_qt_ = np.multiply(rho_temp, diffusive_flux_qt_)
+    total_flux_qt_     = np.multiply(rho_temp, total_flux_qt_)* 1000
```
Comment (trontrytel) [inline tests/les_data/reduce_pycles_netcdf.py]: Why do you multiply by 1000 here?

File: Turbulence_PrognosticTKE.pyx [hunk 3]
```diff
@@ -1739,6 +1745,9 @@ cdef class EDMF_PrognosticTKE(ParameterizationBase):
         with nogil:
             for k in xrange(nz):
                 GMV.U.new[k+gw] = x[k]
+            self.diffusive_flux_u[gw] = interp2pt(Case.Sur.rho_uflux, -rho_ae_K[gw] * dzi *(GMV.U.values[gw+1]-GMV.U.values[gw]) )
+            for k in xrange(self.Gr.gw+1, self.Gr.nzg-self.Gr.gw):
+                self.diffusive_flux_u[k] = -0.5 * self.Ref.rho0_half[k]*ae[k] * self.KH.values[k] * dzi * (GMV.U.values[k+1]-GMV.U.values[k-1])
```
Comment (ilopezgp) [inline Turbulence_PrognosticTKE.pyx]: The coefficient should be KM, not KH, since we are diffusing momentum.

File: Turbulence_PrognosticTKE.pyx [hunk 4]
```diff
@@ -1750,6 +1759,9 @@ cdef class EDMF_PrognosticTKE(ParameterizationBase):
         with nogil:
             for k in xrange(nz):
                 GMV.V.new[k+gw] = x[k]
+            self.diffusive_flux_v[gw] = interp2pt(Case.Sur.rho_vflux, -rho_ae_K[gw] * dzi *(GMV.V.values[gw+1]-GMV.V.values[gw]) )
+            for k in xrange(self.Gr.gw+1, self.Gr.nzg-self.Gr.gw):
+                self.diffusive_flux_v[k] = -0.5 * self.Ref.rho0_half[k]*ae[k] * self.KH.values[k] * dzi * (GMV.V.values[k+1]-GMV.V.values[k-1])
```
Comment (ilopezgp) [inline Turbulence_PrognosticTKE.pyx]: Same here, coefficient should be KM

**Pull request merged**